AutoMaxLair.py

```diff
@@ -169,6 +169,7 @@ def battle(inst) -> str:
             inst.lives -= 1
             if inst.lives != 0:
                 inst.log('The lives counter was not 0.', 'WARNING')
+                inst.lives = 0
```

Comment:
We need to put lives to 0 for two reason : 
- detect the lose and not award a win later on
- calculate the dynite ore gain